src/load.jl

```diff
@@ -11,13 +11,17 @@ Main function to load paths of all subjects in one bids_root folder. Will return
 
 # Keywords
 derivatives (Bool::true): Look for data in the derivatives folder
+
 specificFolder (Union{Nothing,AbstractString}::nothing): Specify a specific folder name in either derivatives or bids_root to look for data.
+
 excludeFolder (Union{Nothing,AbstractString}::nothing): Exclude a specific folder from data detection.
```

Comment:
inconsistency with camelCase under_score.

We kind of settled to `exclude_folder` rather than camelcase. This makes changes necessary pretty much everywhere and will be breaking (but not many people use it yet, so I think shold be fine)